src/preamble.js

```diff
@@ -892,6 +892,11 @@ function getBinaryPromise() {
   });
 }
 
+var wasmSourceMap;
```

Comment:
I think this variable can be inside the ifdef? Less unnecessary code (although the optimizer might remove it if it's unused), but also more clear that it's only needed in that case.

Reply:
I tried that, but then the code cannot detect if the source map is available by checking `wasmSourceMap`.

Reply:
Wait nvm we can just wrap code that accesses `wasmSourceMap` in `#if LOAD_SOURCE_MAP`.

Reply:
It's not already wrapped that way? It should be, since we only use source maps in that case, so if it was missed let's fix that.